ui/src/app/settings/components/repos-list/repos-list.tsx

```diff
@@ -105,18 +120,8 @@ export class ReposList extends React.Component<RouteComponentProps<any>, {connec
                         items: [
                             {
                                 iconClassName: 'fa fa-plus',
-                                title: 'Connect Repo using SSH',
-                                action: () => (this.showConnectSSHRepo = true)
-                            },
-                            {
-                                iconClassName: 'fa fa-plus',
-                                title: 'Connect Repo using HTTPS',
-                                action: () => (this.showConnectHTTPSRepo = true)
-                            },
-                            {
-                                iconClassName: 'fa fa-plus',
-                                title: 'Connect Repo using GitHub App',
-                                action: () => (this.showConnectGitHubAppRepo = true)
+                                title: 'Connect New Repo',
```

Comment:
IMHO, the repo already exists, so perhaps we should not use "New" Repo in case people misinterpret this and think they are creating a new repo and connecting to it.  I think `Connect Repo` is fine. Also, the existing buttons are, "Connect Repo using X" and not "Connect New Repo using X".   Note that the connection is new however, and we're not phrasing it that way. eg. `Create New Connection to Repo`